playwright_test/tests/launchWidgetAndSendMessage.spec.js

```diff
@@ -0,0 +1,61 @@
+
+/*
+ * LaunchWidgetAndSendMessage.spec.js
+ * playwright_test/test
+ * Created by Archit on 20/12/2011. 
+ */
+
+import { test, expect } from '@playwright/test';
+import { widgetLocators, locators } from '../locaterPackage/kmLocators';
+import { url , app_id } from '../utils/kmSecret';
+import { script } from '../utils/kmScript';
+const {chromium} = require('playwright');
```

Comment:
You are explicitly mentioning the chromium browser only, due to this your script will run only on chromium browsers, not in other browsers like firefox and WebKit.

Reply:
no no, if you run tests out side of the tests folder it will run for all 3 browser

Reply:
All 3 browsers are chromium only

Reply:
no, if you check the report it will show you all 3 browsers

Reply:
actually, @abhishek7214 playwright doesn't support parallel multiple browser testing for that we have to change the name of the browser every time before running the test or we have to use 3rd party s/w like BrowserStack to perform that.
for cross-verification, I asked in the playwright community as well.
Am I correct @vipinsonu1 ??

Reply:
This was also fixed in #1002 this PR